### PR TITLE
Added ci configs and ci jobs for Power VS

### PR DESCRIPTION
**Pull request by Karthik-K-N**
This PR will add prow support for PowerVS cloud provider

### CODE REVIEW

**Comment by mkumatag** [comment]
@Karthik-K-N seems like we need go vendoring, lets push that change.

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve

File: core-services/image-mirroring/openshift/mapping_origin_4_11 [hunk 1]
```diff
@@ -145,6 +145,7 @@ registry.ci.openshift.org/origin/4.11:hyperkube quay.io/openshift/origin-hyperku
 registry.ci.openshift.org/origin/4.11:hypershift quay.io/openshift/origin-hypershift:4.11 quay.io/openshift/origin-hypershift:4.11.0 quay.io/openshift/origin-hypershift:latest
 registry.ci.openshift.org/origin/4.11:hypershift-operator quay.io/openshift/origin-hypershift-operator:4.11 quay.io/openshift/origin-hypershift-operator:4.11.0 quay.io/openshift/origin-hypershift-operator:latest
 registry.ci.openshift.org/origin/4.11:ibm-cloud-controller-manager quay.io/openshift/origin-ibm-cloud-controller-manager:4.11 quay.io/openshift/origin-ibm-cloud-controller-manager:4.11.0 quay.io/openshift/origin-ibm-cloud-controller-manager:latest
+registry.ci.openshift.org/origin/4.11:ibm-powervs-cloud-controller-manager quay.io/openshift/origin-ibm-powervs-cloud-controller-manager:4.11 quay.io/openshift/origin-ibm-powervs-cloud-controller-manager:4.11.0 quay.io/openshift/origin-ibm-powervs-cloud-controller-manager:latest
```
Comment (bparees) [inline core-services/image-mirroring/openshift/mapping_origin_4_11:148]: elsewhere we have "powervs" as the prefix for this platform (`origin-powervs-machine-controllers`).

we should be consistent between the two unless something distinct is actually intended by `ibm-powervs` vs `powervs`

Comment (bparees) [reply core-services/image-mirroring/openshift/mapping_origin_4_11:148]: (my sense is that this new image should be `powervs-cloud-controller-manger`)

Comment (Karthik-K-N) [reply core-services/image-mirroring/openshift/mapping_origin_4_11]: Updated the image name to `powervs-cloud-controller-manager`

File: ci-operator/config/openshift/cloud-provider-powervs/openshift-cloud-provider-powervs-main.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: vet
+  commands: go vet $(go list ./... | grep -v "/vendor/")
+  container:
+    from: src
+- as: test
+  commands: go test -v -race -covermode=atomic -coverprofile=cover.out $(go list ./...)
```
Comment (JoelSpeed) [inline ci-operator/config/openshift/cloud-provider-powervs/openshift-cloud-provider-powervs-main.yaml:50]: If you want prow to pick up your code coverage, it needs to be have `coverage` in the name and also be put into the `ARTIFACTS_DIR`, at the moment I don't think the cover.out file will go anywhere

**Comment by bparees** [comment]
image name is approved
/approve

File: ci-operator/config/openshift/cloud-provider-powervs/openshift-cloud-provider-powervs-main.yaml (hunk 2 above)
Comment (mkumatag) [reply ci-operator/config/openshift/cloud-provider-powervs/openshift-cloud-provider-powervs-main.yaml:50]: @JoelSpeed considering the nature of this repo(will get replaced with the IBM Cloud in the future), do you think is this coverage running here is necessary(the patch we are maintains in this repo is trivial one) or gives any value add?

Comment (mkumatag) [reply ci-operator/config/openshift/cloud-provider-powervs/openshift-cloud-provider-powervs-main.yaml:50]: oops, its part of UT test job.. of course we need it but need coverage with this or not is the question :)

Comment (JoelSpeed) [reply ci-operator/config/openshift/cloud-provider-powervs/openshift-cloud-provider-powervs-main.yaml:50]: I'm happy for this to merge as is, I was just working on fixing the coverage on another repo yesterday so thought I'd make a comment. I'm happy to add LGTM

**Pull request merged**